tests/e2e/specs/e2e.spec.ts

```diff
@@ -69,6 +69,13 @@ test.describe("pages", () => {
     await page.getByText("legacy page").click();
     await expect(page.getByText("React Body")).toHaveCount(1);
   });
+
+  test("it redirects trailing slashes with custom routes", async ({ page }) => {
+    await page.goto("./this-is-a-custom-route/");
```

Comment:
for tests we could just reuse existing routes like `/vite-page/` i think?

Reply:
We can't reuse vite-page without adding a `+route.ts` to it. I think we should keep vite-page as a 'normal' page for tests.